### ISSUE TITLE
Android: The blocked login could be escaped by re-importing the same wallet

### ISSUE THREAD

**Issue by vsabolc** [HTMLCOIN/htmlcoin-android#52]
@senagbe

Steps to reproduce:
- Open the Android app
- Tap on Login
- Enter incorrect PIN three times
- Try to log in again
- An error message appears and the login is blocked
- Re-install the app
- Tap on Import Wallet
- Import the same wallet
- New PIN is required

**Comment by senagbe** [comment]
@vsabolc this doesn;t sound like an error to me since if you have the restore password you should be able restore?

**Comment by dohaivu** [comment]
Yes, import wallet always requires a new pin to secure the new wallet.
